demo/angular-editable-text.js

```diff
@@ -10,31 +10,32 @@
 
 })();
 
-/**
- * Created by Gabriel Grinberg on 6/13/14.
- */
+/*
+   Created by Gabriel Grinberg on 6/13/14.
+   */
 
-(function () {
+(function() {
   'use strict';
-  angular.module('gg.editableText')
-    .directive('editableText', ['EditableTextHelper', function (EditableTextHelper) {
+  angular.module('gg.editableText', ['puElasticInput'])
+    .directive('editableText', ['$timeout', 'EditableTextHelper', function($timeout, EditableTextHelper) {
       return {
         scope: {
           editableText: '=',
           editMode: '=',
           placeholder: '@',
-          onChange: '&'
+          onChange: '&',
+          onReject: '&',
```

Comment:
Good point.  Lets see if there's any update from them about the main repo's viability.  If it's abandoned we can still release it, but maybe have this become the primary repo.